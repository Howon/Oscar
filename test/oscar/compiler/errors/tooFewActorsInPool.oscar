message testNow()


actor Tester() {
  
  receive = {
    | testNow() => {
        Println("Error");
    }
  }
}

def main() => unit = {
  pool<Tester> pool1 = spawn pool<Tester>({}, 0);
  message<testNow>() |>> pool1;
}