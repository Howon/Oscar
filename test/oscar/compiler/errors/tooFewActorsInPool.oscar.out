Error: Number of actors in a pool must be at least 1
